src/components/Photos/Photos.js

```diff
@@ -0,0 +1,33 @@
+import { connect } from 'react-redux';
+
+import {useFile} from '../../hooks/useFile'
+
+import './Photos.css';
+
+import heart from '../../assets/svg/heart-solid.svg';
+
+const Photos = (props) => {
```

Comment:
Wykorzystujesz gdzieś to props?

Reply:
Nie wiedziałam co z tym zrobić i czy mam się łączyć ze Store, które już było w projekcie, dlatego narazie zostawiłam.